src/libs/actions/IOU.ts

```diff
@@ -2248,6 +2248,35 @@ function trackExpense(
     Report.notifyNewAction(activeReportID, payeeAccountID);
 }
 
+function getOrCreateOptimisticSplitChatReport(existingSplitChatReportID: string, participants: Participant[], participantAccountIDs: number[], currentUserAccountID: number) {
+    const existingChatReportID = existingSplitChatReportID || participants[0].reportID;
+    let existingSplitChatReport = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${existingChatReportID}`];
+    if (!existingSplitChatReport) {
+        existingSplitChatReport = participants.length < 2 ? ReportUtils.getChatByParticipants(participantAccountIDs) : null;
+    }
+    let newChat: ReportUtils.OptimisticChatReport | EmptyObject = {};
+    const allParticipantsAccountIDs = [...participantAccountIDs, currentUserAccountID];
+    if (!existingSplitChatReport && participants.length > 1) {
+        newChat = ReportUtils.buildOptimisticChatReport(
+            allParticipantsAccountIDs,
+            '',
+            CONST.REPORT.CHAT_TYPE.GROUP,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
+        );
+    }
+    if (isEmptyObject(newChat)) {
```

Comment:
```suggestion
    if (!existingSplitChatReport && isEmptyObject(newChat)) {
```

Reply:
Why do you think we should make this change? I'm not seeing it.

Reply:
I'm gonna rework this. The logic is fine the way it is. But I am realizing that it's confusing and that the `splitChatReport` will always be equal to the `existingSplitChatReport`. Which is very unnecessary and confusing 🤔

Reply:
Fixed this up and added some code comments. Logic seems the same, but mind double checking?

Reply:
the logic is now clear to me. Thank you for the change, Marc 😄